### ISSUE TITLE
indent: class body ignored

### ISSUE THREAD

**Issue by a-x-** [eslint/eslint#11719]
I found closed similar issues: https://github.com/eslint/eslint/issues/4696, https://github.com/eslint/eslint/issues/4372

**environment**

* **ESLint Version:** v5.16.0
* **Node Version:** v11.12.0
* **npm Version:** 6.9.0

**parser**: babel-eslint

**minimal configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
	parser: 'babel-eslint',
	parserOptions: {
		ecmaVersion: 6,
		sourceType: 'module',
		ecmaFeatures: { },
	},

	settings: { },
	rules: {
		'indent': ['warn', 2, { SwitchCase: 1 }]
	},
};
```

</details>

**source code**

```js
// works in functions:
// function Foo () {
// 				cellContentProps = { onStatusChange: this.props.onStatusChange }
// }
// --> warning  Expected indentation of 2 spaces but found 4 tabs

// does not works in class bodies:
export default class Foo {
          cellContentProps = { onStatusChange: this.props.onStatusChange }
}
// --> no warnings
```

```bash
eslint path/to/file.js
```

**expected**
`warning  Expected indentation of 2 spaces but found 4 tabs`

**pull request?** maybe I can create PR, if you describe all the details how and where to fix

**Comment by aladdin-add** [comment]
thanks for the report, I can repro the issue!

**Comment by not-an-aardvark** [comment]
I think this is a duplicate of https://github.com/eslint/eslint/issues/11485.

**Comment by a-x-** [comment]
maybe it is a dup, but problem still actual. I enumered some similar issues too.

**Comment by mysticatea** [comment]
Class fields syntax is not a part of ECMAScript yet. Per [README.md](https://github.com/eslint/eslint#what-about-experimental-features), core rules don't support such an experimental syntax. You can request to plugins that support experimental syntaxes such as `eslint-plugin-babel`.
